tests/conf/oscd_all.yaml

```diff
@@ -11,7 +11,7 @@ experiment:
     in_channels: 26
     num_classes: 2
     num_filters: 1
-    ignore_zeros: True
+    ignore_zeros: False
```

Comment:
We shouldn't ignore_zeros with two classes, this causes the Accuracy metric to break